components/forms/Login/index.jsx

```diff
@@ -0,0 +1,42 @@
+import Link from 'next/link';
+
+const Login = () => {
+  return (
+    <form className="w-80">
+      <div className="h-24 mb-5">
+        <label className="block text-white text-base font-extrabold mb-3" htmlFor="email">
```

Comment:
the `mb-3` makes a 12px margin but the design expect an 8px margin. 
In this specific case, none of the default tailwind classes would fit. So I think we need to customize the default scales, see here: https://tailwindcss.com/docs/margin#customizing-your-theme ;-)

Reply:
![image](https://user-images.githubusercontent.com/77609814/160320754-5d2a9627-1d12-4a9e-a9d4-6d50f6fa63b6.png)

Reply:
actually `mb-2` resolves to `margin-bottom: 2px`, done in https://github.com/tyrro/react-survey/pull/50/commits/42a749da2be9185adad4215698e02f740cb23b2f